arangod/V8Server/v8-analyzers.cpp

```diff
@@ -295,18 +296,28 @@ void JS_Create(v8::FunctionCallbackInfo<v8::Value> const& args) {
   >();
 
   if (!analyzers) {
-    TRI_V8_THROW_EXCEPTION_MESSAGE( // exception
-      TRI_ERROR_INTERNAL, // code
-      std::string("failure to find feature '") + arangodb::iresearch::IResearchAnalyzerFeature::name() + "'" // message
+    TRI_V8_THROW_EXCEPTION_MESSAGE(
```

Comment:
most of this `feature = lookupFeature(); if (!feature)` thing is actually unnecessary, because there is also `getFeature`. getFeature will return a pointer to the feature if it exists or throw if it doesn't. Using that could greatly simplify things in many places.

Reply:
got it, thanks